docs/onboarding/basics/10-ibc.md

```diff
@@ -0,0 +1,12 @@
+# IBC
+
+At Composable, we're investing quite a lot in the Inter-Blockchain-Communication protocol (IBC). This protocol originates from the Cosmos ecosystem and is designed for message passing and token transfers between blockchains. Not only does it work well for Cosmos-based chains, but also for any chain that has deterministic finality. IBC is based on mutual light clients on each chain, which can be used to trustlessly exchange data between the chains.
+
+## The protocol
+
+[![IBC Explained to Cosmonauts Ep. 1](https://img.youtube.com/vi/dYuTYykRhH4/maxresdefault.jpg)](https://youtu.be/dYuTYykRhH4)
+
+## Further Reading
+
+- [IBC](https://ibcprotocol.org/)
```

Comment:
A reference to our own https://medium.com/composable-finance/trustless-bridging-438a6e5c917a

Reply:
Will include!